build/yaml/crd/vpc/crd.nsx.vmware.com_vpcnetworkconfigurations.yaml

```diff
@@ -72,10 +72,22 @@ spec:
                   description: SharedSubnet defines the information for a Subnet shared
                     with vSphere Namespace.
                   properties:
+                    name:
```

Comment:
Can it be defined as immutable property, or add a rule

Reply:
but the Subnets could be updated?